Signal/src/ViewControllers/NewGroupViewController.m

```diff
@@ -606,9 +606,15 @@ - (BOOL)shouldHideLocalNumber
     return YES;
 }
 
-#pragma mark - GroupViewHelperDelegate
+#pragma mark - AvatarViewHelperDelegate
 
-- (void)groupAvatarDidChange:(UIImage *)image
+- (NSString *)avatarActionSheetTitle
+{
+    return NSLocalizedString(
+        @"NEW_GROUP_ADD_PHOTO_ACTION", @"Action Sheet title prompting the user for a group avatar");
```

Comment:
This string is still group-only.  This delegate method is used (among other things) to customize the actionsheet appearance depending on whether you're changing a "group avatar" or a "profile avatar".